Representable/representable
# Homepage Translations

**Changes**
- Adds Spanish translations to the homepage

**to test**
- Go to homepage and switch the language to Spanish
- Confirm that all phrases on the homepage is translated to spanish
